server/cluster/cluster_test.go

```diff
@@ -858,7 +859,7 @@ func checkStaleRegion(origin *metapb.Region, region *metapb.Region) error {
 	e := region.GetRegionEpoch()
 
 	if e.GetVersion() < o.GetVersion() || e.GetConfVer() < o.GetConfVer() {
-		return core.ErrRegionIsStale(region, origin)
+		return errors.Errorf("region is stale: region %v origin %v", region, origin)
```

Comment:
why not use `errRegionIsStale` you define

Reply:
Oh, I don't export `errRegionIsStale` in `core` package. I decide to directly use `errors.Errorf` since this is only a test.

Reply:
We even don't care about what the error is in our test.